src/App.js

```diff
@@ -1,5 +1,131 @@
+import { Console, Random } from '@woowacourse/mission-utils';
+
 class App {
-  async play() {}
+  constructor() {
+    this.carNames = [];
+    this.carNameObjects = [];
+    this.roundCount = 0;
+  }
+
+  async play() {
+    await this.gameStart();
+    this.gameProgress();
+    this.gameOver();
+  }
+
+  async gameStart() {
+    await this.getCarNames();
+    await this.getRoundCount();
+  }
+
+  gameProgress() {
+    const carNameObjects = this.carNames.map((carName) => new Car(carName));
+
+    Console.print('\n실행 결과');
+
+    // round count 만큼 반복문을 돌면서 결과값을 출력
+    for (let i = 0; i < this.roundCount; i++) {
```

Comment:
i++는 에어비앤비 스타일 가이드에서 지양하는 부분이라고 합니다.
[13.6](https://github.com/airbnb/javascript#variables--unary-increment-decrement) Avoid using unary increments and decrements (++, --). eslint [no-plusplus](https://eslint.org/docs/rules/no-plusplus)

> Why? Per the eslint documentation, unary increment and decrement statements are subject to automatic semicolon insertion and can cause silent errors with incrementing or decrementing values within an application. It is also more expressive to mutate your values with statements like num += 1 instead of num++ or num ++. Disallowing unary increment and decrement statements also prevents you from pre-incrementing/pre-decrementing values unintentionally which can also cause unexpected behavior in your programs.

다음 내용을 참고해 주시면 좋을 것 같아요!

Reply:
안그래도 저번에도 비슷한 피드백은 어떤 분에게 들었던 것 같은데 이부분은 꼭 수정하도록 하겠습니당!!